test/PowerShellEditorServices.Test/Session/PowerShellContextTests.cs

```diff
@@ -160,6 +160,7 @@ public async Task CanGetPSReadLineProxy()
             "..",
             "module"
             );
+            System.Console.WriteLine($"Attempting to import {s_bundledModulesPath}");
```

Comment:
Just FYI I am debugging this test on Windows right now, really want to know why it worked everywhere but there.

Reply:
Hah, me too. I think this last change fixed it.

Reply:
I think this PR should be the first to go in. It made sense to bring in the Changes from https://github.com/PowerShell/PowerShellEditorServices/pull/1520 so I'll close that PR as this handles both issues.

Reply:
Ah maybe not, PowerShell 7 | Windows 10 has already failed.

Reply:
macOS:
executingAssembly /Users/runner/work/1/s/test/PowerShellEditorServices.Test/bin/Debug/netcoreapp3.1/Microsoft.PowerShell.EditorServices.Test.dll

Windows: executingAssembly C:\Users\VssAdministrator\AppData\Local\Temp\b5b3251f-c622-45f2-bc10-c8a646797337\b5b3251f-c622-45f2-bc10-c8a646797337\assembly\dl3\e3727c32\64017e66_b772d701\Microsoft.PowerShell.EditorServices.Test.dll

https://stackoverflow.com/questions/8309411/what-is-cache-appdata-local-assembly-dl3

Reply:
Evidently we shouldn't be using GetExecutingAssembly().Location as this can return a Volume Shadow Copy cached version of the DLL. We're using it in 9 other places...

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/DKattan/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/DKattan/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">

</head>

<body link="#0563C1" vlink="#954F72">



Code | File | Line
-- | -- | --
Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location); | test\PowerShellEditorServices.Test.E2E\DebugAdapterProtocolMessageTests.cs | 26
:   Path.GetDirectoryName(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)); | src\PowerShellEditorServices.Hosting\Commands\StartEditorServicesCommand.cs | 302
Assembly.GetExecutingAssembly().Location, | src\PowerShellEditorServices.Hosting\Commands\StartEditorServicesCommand.cs | 344
private static readonly string s_psesBaseDirPath =   Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location); | src\PowerShellEditorServices.Hosting\EditorServicesLoader.cs | 34
Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), | src\PowerShellEditorServices.Hosting\EditorServicesLoader.cs | 39
private static string s_bundledModulesPath =   Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location),   "..", ".."); | src\PowerShellEditorServices\Services\PowerShellContext\PowerShellContextService.cs | 35
Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location); | test\PowerShellEditorServices.Test.E2E\LanguageServerProtocolMessageTests.cs | 36
Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location); | test\PowerShellEditorServices.Test.E2E\LSPTestsFixures.cs | 31
Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location); | test\PowerShellEditorServices.Test.E2E\Processes\PsesStdioProcess.cs | 19



</body>

</html>